packages/dds/tree/src/index.ts

```diff
@@ -124,10 +126,22 @@ export {
 	// Can not be moved to internalTypes since doing so causes app code to throw errors like:
 	// error TS2742: The inferred type of 'Inventory' cannot be named without a reference to '../node_modules/@fluidframework/tree/lib/internalTypes.js'. This is likely not portable. A type annotation is necessary.
 	type AllowedTypes,
+	type AnnotatedAllowedType,
```

Comment:
This section of the exports (see comment above) is for types that could be in internalTypes if not for the mentioned issue. Given that alpha types can't be put in internal types (its public), and nonsystem types shouldn't be put in internal types, none of these really make sense to put in this section.

If you have ideas on how to better document groups of imports in here, feel free to clarify it (though if its a change to a lot of likes, doing it in its own PR would be good), as this current approach hasn't been getting followed very well.

Reply:
Hmm I don't have any ideas but I'll add that part about alpha and nonsystem types not belonging there since I wasn't aware of that..

Reply:
Oh actually, I see now that there's already a section for alpha APIs.

Reply:
It is confusing that there are more normal types at the bottom though so yeah, maybe this should be reorganized a bit. I'll prob try to do so in a separate PR